### ISSUE TITLE
Spyder does not work with a pip installation of CNTK

### ISSUE THREAD

**Issue by machomachopadre** [spyder-ide/spyder#5876]
**Introduction**
When I executed the command "spyder" to launch Spyder from the anaconda command prompt I got an ImportError related to anaconda not being able to find a DLL. When I ran `pip uninstall cntk` Spyder started launching as expected. Is this because of some dependency incompatibility?

I believe somebody else has had the same issue: https://github.com/Microsoft/CNTK/issues/2490.

**What steps will reproduce the problem?**
In the Anaconda3 prompt:
1. Install CNTK: `pip install https://cntk.ai/PythonWheel/GPU/cntk-2.2-cp27-cp27m-win_amd64.whl`
2. Try to launch Spyder: `spyder`

**What is the expected output? What do you see instead?**
The expected output is spyder launching as usual, but what actually happens is an ImportError. The last two lines of the traceback are:
```
from PyQt5.QtWidgets import *
ImportError: DLL load failed: The specified procedure could not be found.
```

**Please provide any additional information below**
I had verified that python3.dll and python36.dll are installed in the Anaconda3 working directory, and I also tried to reset spyder settings to no avail.

Another (possibly) important strange thing I noticed was that when I ran "spyder.exe" from the Anaconda3\Scripts directory, **Spyder launched without any error**.

## Versions and main components
* Spyder Version: 3.2.4
* Python Version: 3.6.3
* PyQt Version: 5.6.0
* Operating system: Windows 10 64x

**Comment by ccordoba12** [comment]
You're installing a wheel for Python 2 in Python 3, so I'd say that's the problem. I know that because of the info part of the wheel name:

    cntk-2.2-cp27-cp27m-win_amd64.whl

`cp27` means that this is for Python 2.7.

So please install a CNTK issue for Python 3.6 (if available) and let us know if that fixes the problem for you.

**Comment by machomachopadre** [comment]
@ccordoba12 sorry, I'd actually given the wrong whl link in the post. I've updated accordingly.

**Comment by ccordoba12** [comment]
And the problem persists?

**Comment by edmondja** [comment]
Yes the problem persists, I have se same problem and I installed the right version.
https://github.com/Microsoft/CNTK/issues/2490
https://github.com/Microsoft/CNTK/issues/2654

**Comment by ccordoba12** [comment]
Thanks for the references @edmondja! Now it's clear that this is not a Spyder problem. However, I'll leave this one open so that it's easier to find for other people.
